### PR TITLE
89 sqrt

### PR DESCRIPTION
**Pull request by levBagryansky**
Closes: #89, #92 
Implemented `number.sqrt`
Tests are already exist.

### CODE REVIEW

**Comment by levBagryansky** [comment]
@EugeneDar please review

File: src/main/eo/org/eolang/math/number.eo [hunk 1]
```diff
@@ -363,11 +363,34 @@
           nan
         *
           TRUE
-          pow.
-            number
-              value
-            0.5
-
+          babylonian-metod
```
Comment (EugeneDar) [inline src/main/eo/org/eolang/math/number.eo:366]: @levBagryansky typo here?

**Review by EugeneDar — APPROVED** [review]
@levBagryansky looks good.

@yegor256 please, have a look.

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**